scrapinghub/client/collections.py

```diff
@@ -91,7 +91,7 @@ def list(self):
 
         :return: a list of collections where each collection is
             represented by a dictionary with ('name','type') fields.
-        :rtype: list[dict]
+        :rtype: list of dicts
```

Comment:
@vshlapakov try this fix https://github.com/tomchristie/django-rest-framework/issues/4882#issuecomment-277995050 (but it worth trying `:class:` prefix first - would be more appropriate)